fortifyapi/client.py

```diff
@@ -584,21 +584,34 @@ def get(self, id):
         with self._api as api:
             return Issue(self._api, api.get(f"/api/v1/projectVersions/{self.parent['id']}/issues/{id}")['data'], self.parent)
 
-    def assign(self, user):
+    def assign(self, id, user, revision=0):
```

Comment:
we are in class `Issue` wouldn't we know the id already, given we call `self.assert_is_instance()`?

Ergo you could do

```python
def assign(self, user, revision=0):
  ...
              "id": self['id'],
```

Reply:
no. for example if an application /version has greater than 1, which id would we be assigning?  We need to be explicit here rather than an implicit id from the instance